qa/L1_ssd_training/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages="numpy torch torchvision mlperf_compliance matplotlib Cython"
+
+pushd ../..
+
+cd docs/examples/pytorch/single_stage_detector/
+
+test_body() {
+    # workaround due to errors while pycocotools is int "pip_packages" above
+    pip install -I pycocotools
+
+    # test code
+    python train.py --data=/data/coco/coco-2017/coco2017/ --epochs=1
```

Comment:
Is it possible to use distributed training here? It will be faster then

Reply:
Not really, this implementation is not prepared for distributed training. I was supposed to add support in next stage of work.